definition/ILivechatBusinessUnit.ts

```diff
@@ -0,0 +1,9 @@
+export interface ILivechatBusinessUnit {
```

Comment:
```suggestion
export interface IOmnichannelBusinessUnit {
```

Reply:
Can we please change this to omnichannel, since by livechat we usually refer to the Livechat widget, but these units will be used across entire omnichannel